cmake/OpenGeode.cmake

```diff
@@ -28,24 +28,20 @@ include("${UTILS_FILE}")
 # Get OpenGeode dependencies
 find_package(Async++ REQUIRED CONFIG NO_DEFAULT_PATH PATHS ${ASYNCPLUSPLUS_INSTALL_PREFIX})
 find_package(Bitsery REQUIRED CONFIG NO_DEFAULT_PATH PATHS ${BITSERY_INSTALL_PREFIX})
+find_package(ghcFilesystem REQUIRED CONFIG NO_DEFAULT_PATH PATHS ${FILESYSTEM_INSTALL_PREFIX})
 find_package(minizip REQUIRED CONFIGS "minizip.cmake" NO_DEFAULT_PATH PATHS ${MINIZIP_INSTALL_PREFIX})
 find_package(nanoflann REQUIRED CONFIG NO_DEFAULT_PATH PATHS ${NANOFLANN_INSTALL_PREFIX})
 find_package(spdlog REQUIRED CONFIG NO_DEFAULT_PATH PATHS ${SPDLOG_INSTALL_PREFIX})
 find_package(Threads REQUIRED)
 
-add_library(filesystem INTERFACE IMPORTED)
-set_property(TARGET filesystem APPEND PROPERTY 
-    INTERFACE_INCLUDE_DIRECTORIES
-        ${FILESYSTEM_INSTALL_PREFIX}
-)
-
 copy_windows_binaries(Async++)
 
 # Install OpenGeode third-parties
 install(
     DIRECTORY
         ${ASYNCPLUSPLUS_INSTALL_PREFIX}/
         ${BITSERY_INSTALL_PREFIX}/
+        ${MINIZIP_INSTALL_PREFIX}/

```

Comment:
Minizip is still static ? If so, no need to install it

Reply:
Yes it is. Nice catch